README.md

```diff
@@ -31,6 +31,13 @@ To override the default migrations path (`db/migrate`), add the following line t
 Mongoid::Migrator.migrations_path = ['foo/bar/db/migrate', 'path/to/db/migrate']
 ```
 
+If you want to use output migration use the hook `after_migrate`
+```
+Mongoid::Migration.after_migrate = Proc.new do |output, name|
```

Comment:
pretty minor but it would be better to use `lambda` instead of `Proc.new`, or the arrow syntax (`->(output, name) {}`) as it provides better isolation than Proc (for example a `return` inside a lambda only exits the lambda). As a rule of thumb, always use lambda unless you really know why you need something else (the arrow syntax is the same as a lambda).